index.md

```diff
@@ -11,16 +11,16 @@ Before you get started
 
 ###  What is Rails?
 
-Rails is a framework for building web applications and it uses the Ruby programming language. The important thing that differentiates a web application from a web-site is that an application includes a database on the backend that allows it to store and return data over long periods of time. It's likely that most web pages you visit these days are actually web applications.
+Rails is a framework for building web applications, and it uses the Ruby programming language. The important thing that differentiates a web application from a website is that an application includes a database on the backend that allows it to store and return data over long periods of time. It's likely that most web pages you visit these days are actually web applications.
 
-Rails is also open source means that no one "owns" the Rails framework. While there's a governing body that helps set standards and ensure that it continues to grow and develop with technology, anyone can contribute to it's development. This also means that Rails will always be free to use.
+Rails is also open source, which means that no one "owns" the Rails framework. While there's a governing body that helps set standards and ensures that it continues to grow and develop with technology, anyone can contribute to its development. This also means that Rails will always be free to use.
```

Comment:
Based on the Chicago Manual of Style, I would argue "yes." 

![img_20170303_000737](https://cloud.githubusercontent.com/assets/2002467/23538975/c4815bf8-ffa5-11e6-8d8e-b99c391e8311.jpg)

Although I would be willing to concede that this use is restrictive and that the sentence needs to be re-written to avoid the "which" altogether. Perhaps "Rails is also open source. This means that no one "owns" the Rails framework."